GeoUK/OSTN02Transform.cs

```diff
@@ -1,110 +0,0 @@
-﻿using GeoUK.Coordinates;
-using GeoUK.Ellipsoids;
-using GeoUK.Projections;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Reflection;
-
-namespace GeoUK
-{
-    public static class OSTN02Transform
```

Comment:
@awelburn don't want to remove currently public API's can you please re-add this.

What was your reasoning for removing it?